tests/e2e/cucumber/features/app-provider/lock.feature

```diff
@@ -12,9 +12,9 @@ Feature: lock
       | Brian |
       | Carol |
     And "Alice" logs in
-    And "Alice" creates the following files into personal space using API
-      | pathToFile | content      |
-      | test.odt   | some content |
+    When "Alice" creates the following resources
+      | resource | type         | content      |
+      | test.odt | OpenDocument | some content |
```

Comment:
this step waits for app-provider to be ready and provide the necessary menu items (new documents and `opein in` option)